website/source/guides/advanced-scheduling/advanced-scheduling.html.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: "guides"
+page_title: "Advanced Scheduling Features"
+sidebar_current: "guides-advanced-scheduling"
+description: |-
+    Introduce advanced scheduling features including affinity and spread.
+---
+
+# Advanced Scheduling with Nomad
+
+The Nomad [scheduler][scheduling] uses a bin packing algorithm to optimize the resource utilization and density of applications in your Nomad cluster. Nomad 0.9 adds new features to allow operators more fine-grained control over allocation placement. This enables use cases similar to the following:
+
+- Expressing preference for a certain class of nodes for a specific application via the [affinity stanza][affinity-stanza].
+
+- Spreading allocations across a datacenter or rack with the [spread stanza][spread-stanza].
```

Comment:
Replace "Spreading allocations across a datacenter or rack".. with "Spreading allocations across a datacenter, rack or any other node attribute or metadata with the [spread stanza].."